File: SearchAndFindBackend/SearchAndFind.DTO/UserRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SearchAndFind.DTO
{
    public class UserRequest : Request
    {
        public string LastName { get; set; }
        public string Mail { get; set; }
        public string Name { get; set; }
        public string Password { get; set; }
        public string DeviceId { get; set; }
        
    }
}

File: SearchAndFind-Frontend/app/src/main/java/uy/com/searchandfind/user/UserRequest.java
package uy.com.searchandfind.user;

import uy.com.searchandfind.Request;

/**
 * Created by strat on 30/09/2017.
 */

public class UserRequest extends Request{

    private String LastName ;
    private String Mail ;
    private String Name ;
    private String Password ;
    private String DeviceId ;

    public UserRequest(){}

    public String getLastName() {
        return LastName;
    }

    public void setLastName(String lastName) {
        LastName = lastName;
    }

    public String getMail() {
        return Mail;
    }

    public void setMail(String mail) {
        Mail = mail;
    }

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        Name = name;
    }

    public String getPassword() {
        return Password;
    }

    public void setPassword(String password) {
        Password = password;
    }

    public String getDeviceId() {
        return DeviceId;
    }

    public void setDeviceId(String deviceId) {
        DeviceId = deviceId;
    }
}
